model.js

```diff
@@ -0,0 +1,16 @@
+const db = require('./db/connection');
+
+
+
+selectTopics = async () => {
+    try {
+        const result = await db.query('SELECT * FROM topics;');
+        return result.rows;
+    }
+    catch (error) {
+        console.log(error)
+        throw error;
+    }
```

Comment:
We don't really need a catch block here as our controller will catch any errors and pass them onto any error handlers we create

Reply:
I thought I always need a try block and catch block when using async await syntax? Or is it okay to have try block by itself?